visiont/scripts/data_gen.py

```diff
@@ -0,0 +1,75 @@
+from tqdm import tqdm
+import numpy as np
+
+from torch.utils.data import DataLoader
+from torchvision.transforms import (
+    Compose,
+    ToTensor,
+    Resize,
+    RandomCrop,
+    RandomHorizontalFlip,
+    RandomApply,
+    RandomGrayscale,
+    ColorJitter,
+    GaussianBlur,
+    RandomRotation,
+    RandomErasing,
+)
+from skimage.io import imsave
+
+from visiont.scripts.train import ImageDirectory, Convert
+
+
+def save_samples(image1, image2, idx, path):
+    def save_tensor_to_image(fpath, img):
+        img = img.squeeze(0)
+        arr = img.cpu().numpy()
+        arr = (np.transpose(arr, (1, 2, 0)) * 255).astype(np.uint8)
+        imsave(fpath.as_posix(), arr)
```

Comment:
Why not

    torchvision.transforms.ToPILImage(mode=None)

Reply:
Also we have einops, say no to raw transpose etc :hugs:

Reply:
Da